images/scripts/debian.setup

```diff
@@ -248,5 +248,7 @@ ln -snf /bin/true /etc/network/if-up.d/openssh-server
 # sudo_root" for details.` message in admins terminal.
 touch /home/admin/.sudo_as_admin_successful
 
-# Pull podman containers
-/var/lib/testvm/podman-images.setup
+# Pull podman containers if we have podman installed
+if command -v podman >/dev/null; then
```

Comment:
I'd have used a more static `if [ "$RELEASE" != "focal" ]` to explicitly rule out 20.04 -- we do expect this to work on all newer and future releases.

Reply:
Yeah, I was thinking that the tests will make sure that the container images are actually there.  But if the tests are also "dynamic" and skip things when images are missing, then we are in trouble.

So I guess I agree that we should be more strict.